### PR TITLE
ci: temporarily remove flaky tests

### PR DESCRIPTION
**Pull request by hkang1**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Review by shiyuann — APPROVED** [review]

File: webui/tests/specs/02-experiments.spec [hunk 1]
```diff
@@ -24,13 +24,13 @@ Specification to test the experiment list page.
   |Kill               |false   |
 * Toggle all table row selection
 
-## Filter experiments by archived
+// ## Filter experiments by archived
```
Comment (RehanSD) [inline webui/tests/specs/02-experiments.spec:27]: Is there a reason we are using `//` instead of `#`?

**Review by RehanSD — COMMENTED** [review]
Looks good to me! Left one quick clarification question, but other than that I think it can be merged!

File: webui/tests/specs/02-experiments.spec (hunk 1 above)
Comment (hkang1) [reply webui/tests/specs/02-experiments.spec:27]: for gauge tests, it uses a special markdown and `#` is treated as a title/section

Comment (RehanSD) [reply webui/tests/specs/02-experiments.spec:27]: Thank you!

**Pull request merged**